coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/teams/DeleteTeamControllerIntegrationTest.java

```diff
@@ -0,0 +1,66 @@
+package io.reflectoring.coderadar.rest.useradministration.teams;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+
+import io.reflectoring.coderadar.graph.useradministration.domain.TeamEntity;
+import io.reflectoring.coderadar.graph.useradministration.domain.UserEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.TeamRepository;
+import io.reflectoring.coderadar.graph.useradministration.repository.UserRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.useradministration.service.security.PasswordUtil;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class DeleteTeamControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private UserRepository userRepository;
+
+  @Autowired private TeamRepository teamRepository;
+
+  private UserEntity testUser;
+
+  private TeamEntity teamEntity;
+
+  @BeforeEach
+  void setUp() {
+    teamEntity = new TeamEntity();
+    teamEntity.setName("testTeam");
+    teamRepository.save(teamEntity, 1);
+
+    userRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.